cmd/brew-bootstrap-jenv-java

```diff
@@ -61,22 +65,42 @@ if ! which jenv &>/dev/null; then
   abort "Error: you need to 'brew install jenv'!"
 fi
 
+if ! [[ -d "$(jenv root)/versions" ]]; then
+  abort_with_jenv_path_message
+fi
+
 if ! jenv version-name &>/dev/null; then
   if ! [[ -z "$JENV_VERSION" ]]; then
     JAVA_REQUESTED="$JENV_VERSION"
   else
     JAVA_REQUESTED="$(jenv local)"
   fi
 
+  if ! [[ "$JENV_VERSION" =~ "." ]]; then
+    warn "Error: requested Java version, \"$JENV_VERSION\", is invalid."
```

Comment:
Hm, it looks like I'm getting the error about containing a period all the time now - it seems like maybe at that point `$JENV_VERSION` is empty?

I get

```
Error: requested Java version, "", is invalid.
hint: Java versions must contain a period.
hint: For example, instead of "11", use "11.0".
```

whether I put in `11.0` or `11`.

I think this is because I'm using a `.java-version` file, so there is no `$JENV_VERSION`. I think instead if this used `$JAVA_REQUESTED` then this would work?